portfolio_project7.html

```diff
@@ -0,0 +1,407 @@
+			<br>
+			<h5 class="  w3-myfont"> ● 只顯示到達站台的名稱、時
+間及路線方向。同時,會顯示
+可轉乘的路線。 </h5>
+			<h5 class=" w3-myfont"> ●This diagram only shows up
+current station / node name,
+time and route continue
+direction.
+Also, show up the
+interchangeable route.</h5>
+
+		</div>
+
+	<hr>
+	
+	</div>
+		<hr>
+	 <div class="w3-row">
+
+	 <h4 class="w3-center  w3-myfont">【專案七 製作方法及完成圖】 </h4><br>
+		
+    <div class="w3-col m6 w3-center w3-padding-large">
+	
+
+	
+
+		
+      <p><b><i class="fa fa-user w3-margin-right"></i>作品 完成範例(Project Concept Art)</b></p>
+      <br>
+      <iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PL9zygzz0hZalVlwxz9AsujtsrcMJpAP6S" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+	</div>
+    
+    <!-- Hide this text on small devices -->
+    <div class="w3-col m6 w3-hide-small w3-padding-large">
+      <h4 class=" w3-myfont">【向量圖繪製 及 加上文字及圖層 使用】</h4>
+        <h5 class=" w3-myfont">  ● </h5>
+		<h5 class="  w3-myfont"> ● </h5>
+		<h5 class="  w3-myfont"> ● </h5>	
+		<h5 class="  w3-myfont"> ●</h5>
+	<br>
+	</div>
+	
+	<div class="w3-col m6 w3-hide-small w3-padding-large">
+      <h4 class=" w3-myfont">【反覆播放的「動畫 」】</h4>
+        <h5 class=" w3-myfont">  ● </h5>
+		<h5 class="  w3-myfont"> ● </h5>
+		<h5 class="  w3-myfont"> ● </h5>	
+		<h5 class="  w3-myfont"> ●</h5>
+	<br>
+	</div>
+
+  </div>
+	
+	
+	<!--
+    <div class="w3-row w3-myfont"  id="proj4_photo">
+        <div class="w3-col m6 w3-center w3-padding-large">
+			<p><b><i class="fa fa-user w3-margin-right"></i>全線地圖</b></p>
+			<img src="pictures\media_archive_page\project_4_6_screenshot\mchouse1.jpg" onclick="onClick(this)" class="w3-round w3-image w3-opacity w3-hover-opacity-off" alt="Testing1" width="600" height="500">
+        </div>
+        <div class="w3-col m6 w3-center w3-padding-large">
+			<p><b><i class="fa fa-user w3-margin-right"></i>略版地圖</b></p>
+			<img src="pictures\media_archive_page\project_4_6_screenshot\mchouse3.jpg" onclick="onClick(this)" class="w3-round w3-image w3-opacity w3-hover-opacity-off" alt="Testing1" width="600" height="500">
+        </div>
+    </div>
+	<div class="w3-row w3-myfont"  id="proj4_photo">
+        <div class="w3-col m6 w3-center w3-padding-large">
+			<p><b><i class="fa fa-user w3-margin-right"></i>到達站台圖片</b></p>
+			<img src="pictures\media_archive_page\project_4_6_screenshot\mchouse2.png" onclick="onClick(this)" class="w3-round w3-image w3-opacity w3-hover-opacity-off" alt="Testing1" width="600" height="500">
+        </div>
+    <div class="w3-col m6 w3-center w3-padding-large">
+			<p><b><i class="fa fa-user w3-margin-right"></i>遊戲內 數值/範圍預覽</b></p>
+			<img src="pictures\media_archive_page\project_4_6_screenshot\mchouse4.png" onclick="onClick(this)" class="w3-round w3-image w3-opacity w3-hover-opacity-off" alt="Testing1" width="600" height="500">
+        </div>
+    </div>
+	
+-->

```

Comment:
### Change to Release status/methodology

- [ ] Release (Youtube)
- [ ] Methodology